create-database.sql

```diff
@@ -1,53 +1,50 @@
 CREATE DATABASE ShopDB;
+
 USE ShopDB;
 
+
 CREATE TABLE Countries (
-    ID INT,
-    Name VARCHAR(50),
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(50) NOT NULL,
     PRIMARY KEY (ID)
 );
 
 CREATE TABLE Products (
     ID INT AUTO_INCREMENT,
```

Comment:
When inserting data into tables with `AUTO_INCREMENT` fields, such as `Products`, it's best practice to omit the ID values. The database will automatically handle these, ensuring unique and sequential IDs.